migration/migrate_db.go

```diff
@@ -0,0 +1,93 @@
+package main
```

Comment:
to my understanding talking to @tirthyme, the intention here is to have this as a standalone binary/executable. This code is not meant to be exported and consumed by other packages. The use case is to do something like this from the Makefile:

```shell
cd hacks/migration; go run migrate_db.go
```

To do that, the package has to be `main`. It's a common pattern to create isolated files for your code:

```shell
/ (root)
 -> examples (dir)
      -> ex1 (dir)
            ex1.go (main pkg)
      -> ex2 (dir)
             ex2.go (main pkg)
      -> ex3 (dir)
             ex3.go (main pkg)
```

Ideally, we can introduce a `cmd` package for the CLI stuff. In that case, yes it could name the package `db` because it would have been imported by the `cmd` one. Right now we have just the make file where is the common denominator for calling all of these _hacks_.